tasks/install-package.yml

```diff
@@ -0,0 +1,114 @@
+- name: install the rpm repo's key
+  rpm_key:
+    state: present
+    key: "{{ rpm_repo_key }}"
+  when: ansible_os_family == "RedHat"
+
+- name: install apt repo keys
+  apt_key:
+    state: present
+    url: "{{ item.url }}"
+    id: "{{ item.id }}"
+  when: ansible_os_family == "Debian"
+  loop:
+  - { url: "{{ deb_repo_key }}", id: "{{ deb_repo_key_id }}" }
+  - { url: "https://deb.nodesource.com/gpgkey/nodesource.gpg.key", id: "1655A0AB68576280" }
+
+- name: install the rpm repo
+  package:
+    name: "{{ rpm_repo_url }}"
+    state: present
+    disable_gpg_check: "{{ disable_gpg_check_rpm_repo | bool }}"
```

Comment:
If this disables GPG check on repo RPM, those RPMs are not GPG signed, so might not be necessary though I'm sure doesn't hurt either.

Reply:
Yea the default is to gpg check, so I had to add this to stop checking.